phpunit/class-block-comment-filter-test.php

```diff
@@ -0,0 +1,52 @@
+<?php
+/**
+ * Unit tests for block comment filter functions.
+ *
+ * @package Gutenberg
+ * @since 6.9.0
```

Comment:
```suggestion
 * @package Gutenberg
```

The since tag should be added after the feature is stable and a core version has been decided.